bin/health-check.rb

```diff
@@ -7,12 +7,17 @@
 conjur_api = ConjurClient.api
 
 begin
-  if ConjurClient.version == 5
-    conjur_api.resources limit: 5
-  else
-    conjur_api.resource("#{ConjurClient.account}:user:admin").exists?
-  end
-  
+  resource_id =
+    if !ConjurClient.login_host_id.nil?
+      "#{ConjurClient.account}:host:#{ConjurClient.login_host_id}"
+    else
+      "#{ConjurClient.account}:user:#{ConjurClient.authn_login}"
+    end
+
+  # This will throw an exception if the creds are invalid
+  resource_exists = conjur_api.resource(resource_id).exists?
+  raise StandardError.new("Host identity not privileged to read itself") unless resource_exists || ConjurClient.login_host_id.nil?
```

Comment:
This line is so long that it's probably worth breaking down

```
unless resource_exists ...
  raise ...
end
```

Reply:
You could probably do away with the `resource_exists` helper if you break the line down too.